src/java.base/windows/native/libnio/ch/FileKey.c

```diff
@@ -30,32 +30,21 @@
 #include "nio_util.h"
 #include "sun_nio_ch_FileKey.h"
 
-static jfieldID key_volumeSN;    /* id for FileKey.dwVolumeSerialNumber */
-static jfieldID key_indexHigh;   /* id for FileKey.nFileIndexHigh */
-static jfieldID key_indexLow;    /* id for FileKey.nFileIndexLow */
-
-
-JNIEXPORT void JNICALL
-Java_sun_nio_ch_FileKey_initIDs(JNIEnv *env, jclass clazz)
-{
-    CHECK_NULL(key_volumeSN = (*env)->GetFieldID(env, clazz, "dwVolumeSerialNumber", "I"));
-    CHECK_NULL(key_indexHigh = (*env)->GetFieldID(env, clazz, "nFileIndexHigh", "I"));
-    CHECK_NULL(key_indexLow = (*env)->GetFieldID(env, clazz, "nFileIndexLow", "I"));
-}
-
-
 JNIEXPORT void JNICALL
-Java_sun_nio_ch_FileKey_init(JNIEnv *env, jobject this, jobject fdo)
+Java_sun_nio_ch_FileKey_init(JNIEnv *env, jclass clazz, jobject fdo,
+    jintArray finfo)
```

Comment:
Minor nit, no need for the line break in the function declaration.

Reply:
Removed in [eafff23](https://github.com/openjdk/jdk/pull/20429/commits/eafff235646c70d72952c40928776b69c1bb33c9).